src/contexts/GameContext.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {
```

Comment:
I would put this file in the Arcade folder since it's used there. It won't be used elsewhere

Reply:
Yeah, let me revaluate this component and leverage the react-chrome-dino repo. Why attempt to reinvent the wheel when we can simply update their assets/svgs and all that. 👍